### PR TITLE
PromCon 2017 recap blog post

### PR DESCRIPTION
**Pull request by juliusv**
This is very similar to last year's blog post (http://localhost:3000/blog/2016/09/04/promcon-2016-its-a-wrap/), which I think is fine because we're not aiming for maximum originality here.

### CODE REVIEW

**Comment by juliusv** [comment]
@RichiH @grobie

**Comment by juliusv** [comment]
We might wanna integrate / link to the istream pictures somehow... they lack context, so I think for the article itself, the embedded tweets are better, but maybe link to a separate gallery?

File: content/blog/2017-09-04-promcon-2017-recap.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">You like Prometheus but have a need for long-term storage? <a href="https://twitter.com/pauldix">@pauldix</a> presents how <a href="https://twitter.com/InfluxDB">@InfluxDB</a> integrates with Prometheus at <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/hpsO95mPVL">pic.twitter.com/hpsO95mPVL</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898476044327497728">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Several Prometheus core developers also spoke about best practices and new features and developments in Prometheus:
```
Comment (brian-brazil) [inline content/blog/2017-09-04-promcon-2017-recap.md]: The social aspects and staleness talks should go in here too.

File: content/blog/2017-09-04-promcon-2017-recap.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+PromCon 2017 would not have been possible without the help of its sponsors,
+speakers, attendees, and organizers. Thanks so much to all of you! Our Diamond
+and Venue sponsors deserve a special mention at this point, since they did
+the most to support us and made all the food, drinks, video recordings, and
+swag possible:
+
+<h3>Diamond</h3>
+<div class="sponsor-logos">
+  <a href="https://www.weave.works/"><img src="/assets/blog/2017-09-04/weave_logo.png"/></a>
```
Comment (brian-brazil) [inline content/blog/2017-09-04-promcon-2017-recap.md]: These should be in some sane order.

Have you checked the sizes are reasonable?

Comment (juliusv) [reply content/blog/2017-09-04-promcon-2017-recap.md]: The GitLab logo always takes up a bit more vertical space, but not sure what to do about that. I can try to experiment with the layout here a bit more...

What is a sane order (besides solving layout issues)?

File: content/blog/2017-09-04-promcon-2017-recap.md (hunk 1 above)
Comment (juliusv) [reply content/blog/2017-09-04-promcon-2017-recap.md]: I was worried about this section getting too long, but happy to make it longer if you think that's better...

File: content/blog/2017-09-04-promcon-2017-recap.md (hunk 2 above)
Comment (brian-brazil) [reply content/blog/2017-09-04-promcon-2017-recap.md]: Sorting alphabetically is probably easiest.

File: content/blog/2017-09-04-promcon-2017-recap.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Our own <a href="https://twitter.com/juliusvolz">@juliusvolz</a> gives a lecture on best practices and pitfalls when starting with <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a> <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/gTNrxeaxK7">pic.twitter.com/gTNrxeaxK7</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898098812761362432">August 17, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">. <a href="https://twitter.com/mxinden">@mxinden</a> from <a href="https://twitter.com/coreos">@coreos</a> talks about how and why the alertmanager got it&#39;s shiny new UI. <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/zXaZS9rXqc">pic.twitter.com/zXaZS9rXqc</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898525224924327936">August 18, 2017</a></blockquote>
```
Comment (brian-brazil) [inline content/blog/2017-09-04-promcon-2017-recap.md]: its, as it is possessive.

Comment (juliusv) [reply content/blog/2017-09-04-promcon-2017-recap.md]: Yes, but it's a tweet which we can't change - changing the copy of it here doesn't seem good.

**Comment by juliusv** [comment]
@brian-brazil Changed sponsors to alphabetical order, which coincidentally also looks best optically. I also added staleness and social talks.

**Comment by RichiH** [comment]
It feels weird to have more tweets than content, but that's probably me being old :)

Looks fine in the fixup version.

**Comment by juliusv** [comment]
@RichiH Meh - I cannot write much more prose now, and the tweets add content + pictures in an easy way :)

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**